rmw/test/test_types.cpp

```diff
@@ -26,3 +26,82 @@ TEST(test_types, zero_initialized_message_info) {
 
   EXPECT_FALSE(info.from_intra_process);
 }
+
+TEST(test_types, time_equal) {
+  {
+    rmw_time_t a {100, 100};
```

Comment:
@emersonknapp nit: why the extra space i.e.:
```suggestion
    rmw_time_t a{100, 100};
```
?

Reply:
I find it more readable this way but I'm happy to remove the space - the linter doesn't seem to have an opinion either way